### ISSUE TITLE
Prayer Times Incorrect In London

### ISSUE THREAD

**Issue by abduljamac** [jalalmostafa/prayertimes#15]
First of all, thank you so much for creating such a useful extension, I'm on my laptop a lot and this extension is a good reminder to pray on time. 

I just wanted to say the prayer times for London is off compared to https://islamchannel.tv/.

<img width="1247" alt="Screenshot 2021-02-03 at 11 16 50" src="https://user-images.githubusercontent.com/31995285/106739693-54e98780-6611-11eb-8bd5-5ecb029c6aa7.png">

I was wondering if you could get it more accurate, I've tried all calculations available and none of them gives the correct time.

**Comment by jalalmostafa** [comment]
Hi.
Thank you for your feedback! I am happy you like it.

What calculation method are you using? and What islamchannel.tv calculation method is using?
If you go to the extension settings (press on the gear), you can modify the calculation method.

**Comment by abduljamac** [comment]
In the screenshot Im using the calculation "Muslim World League". However, Ive tried all the available calculations and none of them seem to match London prayer times found in islamchannel and https://www.eastlondonmosque.org.uk/prayer-times which I've used for years.

Sadly, Im not able to tell you what calculations islamchannel use but I was able to find this https://www.eastlondonmosque.org.uk/prayer-times-and-calendar-explained from the east london mosque website.

Also I've noticed the alarms don't work, while using it today.
